### ISSUE TITLE
Not displaying full article

### ISSUE THREAD

**Issue by alleyoopster** [Etuldan/spaRSS#267]
With Show Full Content checked when reading an article I often need to toggle check box off and back on from article menu to view full content.

**Comment by ildar** [comment]
That's because app couldn't load the full content for some reason (maybe
internet failure or alike)
There have been a similar issue, phase search.

**Comment by alleyoopster** [comment]
That makes sense, although it is happening a lot to me and it is cumbersome to click on menu to open, click on check box to remove check and click again to see each article that fails. Can I make a suggestion to include a refresh option, maybe pull down to refresh or refresh button?

**Comment by ildar** [comment]
Disagree. It's better fixing the problem:
1. Uncheck the checkbox if failed to load
2. Retry loading on next update (auto or manual)

**Comment by alleyoopster** [comment]
Unchecking the box and leaving the article back to list and then returning to the article doesn't make for a very fluid reading experience especially if reading many articles and sliding through to the next one.

**Comment by ildar** [comment]
Maybe it's a good idea to fetch full content on opening the article? Must
be opt in though.

**Comment by alleyoopster** [comment]
Yes, that would work, but only if needed otherwise I see it lagging as it refreshes each time whether article has full content or it's missing.

**Comment by moshpirit** [comment]
When you add a source you can select if you want to download the entire text or not but there's a very annoying bug with it
